source/plugins/go/src/oms.go

```diff
@@ -87,9 +87,9 @@ const WindowsContainerLogPluginConfFilePath = "/etc/amalogswindows/out_oms.conf"
 const IPName = "ContainerInsights"
 
 const defaultContainerInventoryRefreshInterval = 60
-
 const kubeMonAgentConfigEventFlushInterval = 60
 const defaultIngestionAuthTokenRefreshIntervalSeconds = 3600
+const containerLogV2DCRInterval = 5
```

Comment:
rename to "containerLogV2PollingIntervalSeconds"

Reply:
changed